apoena.html

```diff
@@ -3,26 +3,21 @@
 <head>
   <title>Sample Page</title>
 
-  <script type='text/javascript' src='../src/apoena.js'></script>
-  <script type='text/javascript' src='../src/apoena_parser.js'></script>
-  <script type='text/javascript' src='../src/sample.js'></script>
-  <style type="text/css">
-    canvas { border: 1px solid black; }
-    .todo { border: 1px solid black; background: rgb(230,230,230); width: 800px; padding-left: 10px; }
-    h2 { margin-bottom: 0px; }
-  </style>
+  <script type='text/javascript' src='src/sample.js'></script>
```

Comment:
For safety and better understanding, sample.js should be loaded after jammed.js, as it uses its functions and methods.